fdbserver/workloads/AtomicOpsApiCorrectness.actor.cpp

```diff
@@ -84,6 +84,9 @@ struct AtomicOpsApiCorrectnessWorkload : TestWorkload {
 		case 7:
 			TEST(true); //Testing atomic Add
 			return testAdd(cx->clone(), this);
+		case 8:
```

Comment:
you need to adjust the opType to be able to choose the number 8, (opType = sharedRandomNumber % 9)